understanding/22/focus-not-obscured-minimum.html

```diff
@@ -31,6 +31,8 @@ <h2>Intent of Focus Not Obscured (Minimum)</h2>
 
       <p>Another form of obscuring can occur where light boxes or other semi-opaque effects overlap the item with focus. While less than 100 percent opacity is not causing the component to be <q>entirely obscured</q>, such semi-opaque overlaps may cause a failure of <a href="../understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a>. When a focus indicator can be covered by a semi-opaque component, the ability of the focus indicator to pass 1.4.11 should be evaluated (and pass) while the focus indicator is under the semi-opaque component. The intention in both situations is that the component receiving focus should never be obscured to the point a user cannot tell which item has focus.</p>
 
+      <p class="note">This criterion evaluates the focused <em>component</em>, rather than the focus <em>indicator</em>. The component itself does not include the focus indicator when checking that "the component is not entirely hidden" - unless the focus indicator is inside the component, or focus is indicated by a change to the component itself. Although users benefit from both the component and the focus indicator (if external to the component) not being obscured when tracking the focus, for the purposes of this criterion only checking the component provides a clearer metric. However, if the focus indicator is obscured, it will already fail <a href="focus-visible">2.4.7 Focus Visible</a>.</p>
```

Comment:
Suggested by Giacomo:

```suggestion
      <p class="note">This criterion evaluates the focused <em>component</em>, rather than the focus <em>indicator</em>. The component itself does not include the focus indicator when checking that "the component is not entirely hidden" - unless the focus indicator is inside the component, or focus is indicated by a change to the component itself. Although users benefit from both the component and the focus indicator (if external to the component) not being obscured when tracking the focus, for the purposes of this criterion only checking the component provides a clearer metric. However, if the focus indicator is obscured, it would likely fail <a href="focus-visible">2.4.7 Focus Visible</a>.</p>
```